src/OpenTelemetry/Metrics/MetricTypeExtensions.cs

```diff
@@ -28,6 +28,7 @@ public static class MetricTypeExtensions
         internal const MetricType METRIC_TYPE_GAUGE = (MetricType)0x20;
         /* internal const byte METRIC_TYPE_SUMMARY = 0x30; // not used */
         internal const MetricType METRIC_TYPE_HISTOGRAM = (MetricType)0x40;
+        internal const MetricType METRIC_TYPE_SUM_NON_MONOTONIC = (MetricType)0x80;
```

Comment:
Consider changing this to `METRIC_TYPE_NON_MONOTONIC_SUM`, and update Ln27 to `METRIC_TYPE_MONOTONIC_SUM`.